src/test/java/com/rednavis/vaadin/showcase/backend/jms/JmsTest.java

```diff
@@ -0,0 +1,32 @@
+package com.rednavis.vaadin.showcase.backend.jms;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.rednavis.vaadin.showcase.jms.MessageReceiver;
+import com.rednavis.vaadin.showcase.jms.MessageSender;
+import javax.jms.JMSException;
+import org.junit.jupiter.api.Test;
+
+class JmsTest {
+
+  private MessageSender sender;
+  private MessageReceiver receiver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'receiver' could be replaced by a local variable.](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4992471)